Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -109,14 +120,48 @@ Most of the steps for this phase have been automated using the `finish_release`
+   * In case there are changes other than updates to app and build versions after merging the changes from `develop`, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned. In current workflow every change integrated to release branch is supposed to be go into `develop` soon after, so there shouldn't be any changes. Any differences might be a result of resolving conflicts, or a hotfix PR not being merged previously to `develop`.
    * Set the _Merge_ label once all the required reviewers have approved it.
 1. Update this document if any steps during the release process have changed.
 
 
-## 3. SDK Release
+## Hotfix Release
+
+### Phase 1: Initiation
+
+The hotfix is cut from the latest stable/hotfix release tag (e.g. `4.1.1` from `4.1.0`, or `4.1.2` from `4.1.1`).
+
+**⚠️ NOTE:** Only run the automated process if the previous release has been finished. In order to run the CRP, the GitHub release need to exist, otherwise it will include previous, released tickets.
+
+1. Create the hotfix release branch (e.g. `release/babylon/4.1.1`) from the last released tag and publish it
+1. Create a release channel on Slack (e.g. `ios_release_4_1_1`)
+1. Create a PR (e.g. `[CRP-XXX] Babylon Release 4.1.1`), assign the release engineers, and add the `WIP` label
+1. Trigger the App Center build from that branch using its command (e.g. `/appcenter Babylon branch:release/babylon/4.1.1`) in `#ios-build`.
+1. Trigger the full UI automation run by issuing the command `/stevenson ui_tests branch:release/babylon/4.1.1` in `#ios-build`.
+1. Run CRP `/crp ios release/babylon/4.1.1` in `#ios-launchpad`
+	* **⚠️ NOTE:** Only run the CRP after the previous release has been finished. In order to run the CRP, the GitHub release/tab need to exist, otherwise it will include unrelated tickets.
+
+2. Join the Slack channel (e.g. `ios_release_4_1_1`) to discuss anything related to this release.
+3. Bump the release version by triggering the Slack command (e.g. `/testflight Babylon version:4.1.0`) in `#ios-build` (you can run the command every time you want to upload a new build).
+   * This creates a TestFlight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
+4. Create a new version in [AppStoreConnect](https://appstoreconnect.apple.com) (login using your own account) / My Apps
+  1. On the sidebar click `+ Version or Platform` and select `iOS`.
+  1. Input the new version number.
+  * Note: this can be only done after the previous release has been completed
+
+During this stage, the **release manager** has the following tasks:
+
+1. Ensure the CRP ticket is up-to-date
+     * The CRP ticket is created automatically for iOS during release cutoff ([see details here](https://github.com/babylonhealth/babylon-ios/blob/develop/Documentation/Process/Release%20process/CRP-Bot.md))
```

Comment:
Well if we do all the steps (creating the branch, the PR, etc) for the hotfixes manually as is suggested here, the CRP won't be created automatically. It's only triggered as part of the `release_cutoff` lane/script.

The manual instructions above already lists in the last step to trigger `/crp` manually. so better be consistent, either mention here that "The CRP ticket was created by the last step when you triggered `/crp` …", or modify the instructions to use `/stevenson release_cutoff …` for hotfixes too if that works, and then the CRP will indeed be created automatically.

Reply:
adding the automated command, that should resolve the wording